app/controllers/subscriptions_controller.rb

```diff
@@ -6,4 +6,17 @@ class SubscriptionsController < ApplicationController
   def new
     authorize Subscription
   end
+
+  def create
+    authorize Subscription
+    iugu = session[:iugu]
```

Comment:
How to have access to the `iugu` object that was created at `payments_controller#new ` ?